openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/OpenShiftAssistant.java

```diff
@@ -30,6 +36,10 @@
 
     private OpenShiftAssistantDefaultResourcesLocator openShiftAssistantDefaultResourcesLocator;
 
+    private List<ParameterValue> pvs = new ArrayList<>();
```

Comment:
Can you rename to `parameterValueList`? Because here it is clear but then when I read the code I need to think what it means `pvs` )

Reply:
why `List`? `parameterValues` won't work?